src/core/MOM_open_boundary.F90

```diff
@@ -3877,7 +3877,7 @@ subroutine update_OBC_segment_data(G, GV, US, OBC, tv, h, Time)
   endif
 
   if (OBC%number_of_segments >= 1) then
-    call thickness_to_dz(h, tv, dz, G, GV, US)
```

Comment:
I don't understand what this wider halo size is doing for us, because I think that the call to `pass_var(dz, ...)` on the next lie should have taken care of the halos for dz.  Is it possible that we are working with uninitialized values in land points or something like that?